src/CLI/clitree/cli-xml/mac.xml

```diff
@@ -0,0 +1,328 @@
+                            ptype="LAG_ID"
+                            >
+                        </PARAM>
+                    </PARAM>
+                </PARAM>
+            </PARAM>
+        </PARAM>
+    <ACTION>
+        if test "${s-subcommands}" = "address"; then&#xA;
+            python $SONIC_CLI_ROOT/sonic-cli-mac.py get_openconfig_interfaces_interfaces show_mac.j2 static address ${mac-addr}&#xA; 
+        elif test "${s-subcommands}" = "vlan"; then&#xA;
+            python $SONIC_CLI_ROOT/sonic-cli-mac.py get_openconfig_interfaces_interfaces show_mac.j2 static vlan ${vlan-id}&#xA; 
+        elif test "${s-subcommands}" = "interface"; then&#xA;
+            if test "${interface-subcommand}" = "port-channel"; then&#xA;
+                python $SONIC_CLI_ROOT/sonic-cli-mac.py get_openconfig_interfaces_interfaces show_mac.j2 static interface Port-channel${port-channel-id}&#xA;
+            else&#xA;
+                python $SONIC_CLI_ROOT/sonic-cli-mac.py get_openconfig_interfaces_interfaces show_mac.j2 static interface Ethernet${phy-if-id}&#xA;
+            fi&#xA;
+        else&#xA;
+            python $SONIC_CLI_ROOT/sonic-cli-mac.py get_openconfig_interfaces_interfaces show_mac.j2 static ${__full_line}&#xA;
+        fi
+    </ACTION>
+    </COMMAND>
+
+<!--====================show mac address-table dynamic commands ======================-->
+    <COMMAND
+        help="mac address-table for dynamic commands"
+        name="show mac address-table dynamic">
+        <PARAM
+            name="d-subcommands"
+            help="Dynamic subcommands"
+            mode="switch"
+            ptype="SUBCOMMAND"
+            optional="true"
+            >
+            <PARAM
+                name="address"
+                help="Mac address of dynamic interface"
+                ptype="SUBCOMMAND"
+                mode="subcommand"
+                >
+                <PARAM
+                    name="mac-addr"
+                    help="48 bit mac-address"
+                    ptype="MAC_ADDR"
+                    >
+                </PARAM>
+            </PARAM>
+            <PARAM
+                name="vlan"
+                help="VLAN ID"
+                ptype="SUBCOMMAND"
+                mode="subcommand"
+                >
+                <PARAM
+                    name="vlan-id"
+                    help="VLAN ID"
+                    ptype="VLAN_ID"
+                    >
+                </PARAM>
+            </PARAM>
+            <PARAM
+                name="interface"
+                help="Dynamic Interface ethernet or port-channel"
+                ptype="SUBCOMMAND"
+                mode="subcommand"
+                >
+                <PARAM
+                    name="interface-subcommand"
+                    help="Dynamic interface subcommands"
+                    ptype="SUBCOMMAND"
+                    mode="switch"           
+                    >
+                    <PARAM
+                        name="ethernet"
```

Comment:
Could we represent physical interface just like how it's been represented in Kernel? For example: ethernet -> Ethernet, and vlan -> Vlan, port-channel should be PortChannel?

Reply:
Addressed.